server/vitals.js

```diff
@@ -140,28 +140,33 @@ async function checkButtonBatteries() {
   }
 }
 
-/* 
-logClientMessage expects an object containing key-values that represent clients with recently disconnected and/or reconnected button(s)
-Each key is a clientID that holds an object with the client information and arrays of any disconnected and/or reconnected buttons
-The function will compile all this information into one message per client, and log the message to Sentry
-*/
-function logClientMessage(clientButtonStatusChanges) {
-  // Loop through each client to create the log message
+// Send button status changes as text messages to every client that has them.
+// This function expects an object (clientButtonStatusChanges) with client IDs as keys, and each each value an object with members:
+// - client: The client object
+// - disconnectedButtons: An array of button display names that refer to buttons that have recently disconnected
+// - reconnectedButtons: An array of button display names that refer to buttons that have recently reconnected
+function sendClientButtonStatusChanges(clientButtonStatusChanges) {
+  // Loop through each client to create the Twilio messages
   Object.values(clientButtonStatusChanges).forEach(({ client, disconnectedButtons, reconnectedButtons }) => {
-    const clientDisplayName = client.displayName
-    let buttonLogMessage = ''
+    let message = [i18next.t('buttonStatusChangeStart', { lng: client.language, clientDisplayName: client.displayName })]
 
     if (disconnectedButtons.length > 0) {
-      const buttonNames = disconnectedButtons.sort().join(', ')
-      buttonLogMessage += ` The following buttons have been disconnected: ${buttonNames}.`
+      const buttonDisplayNames = disconnectedButtons.sort().join(', ') // sorted alphabetically
+      message.push(i18next.t('buttonStatusChangeDisconnected', { lng: client.language, buttonDisplayNames }))
     }
 
     if (reconnectedButtons.length > 0) {
-      const buttonNames = reconnectedButtons.sort().join(', ')
-      buttonLogMessage += ` The following buttons have been reconnected: ${buttonNames}.`
+      const buttonDisplayNames = reconnectedButtons.sort().join(', ') // sorted alphabetically
+      message.push(i18next.t('buttonStatusChangeReconnected', { lng: client.language, buttonDisplayNames }))
     }
 
-    helpers.logSentry(`Button status change for: ${clientDisplayName}.${buttonLogMessage}`)
+    // join the message parts with spaces
+    message = message.join(' ')
```

Comment:
I'm imagining that this will be hard to read when there are a lot of status changes. What does this look like on a phone? I wonder if joining with two newlines would be easier to read.

Reply:
That's a good point! I think it will look very strange for bilingual clients, though. Right now, I've worked it out to look as the existing translations:
```
(French translation of full message)
---
(English translation of full message)
```

But if there are several lines in the full message, then it could get funky:

```
(French text here)

(More French text here)
---
(English text here)

(More English text here)
```

It would be easy to replace the spaces with newlines, though. Thoughts?